classes/order/OrderState.php

```diff
@@ -162,4 +162,24 @@ public function isRemovable()
     {
         return !($this->unremovable);
     }
+
+    /**
```

Comment:
Don't forget to describe your method and the parameters. Describing helps other people understand the intent (what the method is for, what is expected in all parameters). In addition, you'll be surprised about how often you will find out naming or design issues just by taking the time to describe stuff.